### PR TITLE
Changing the meta data on key pages to improve SEO

### PR DESCRIPTION
**Pull request by sfshaza2**

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

File: src/docs/get-started/install/index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title: Install
-description: How to set up your code editor.
+description: Install Flutter and get started. Downloads available for windows, macOS, Linux and Chrome OS operating systems.
```
Comment (kwalrath) [inline src/docs/get-started/install/index.md:3]:
```suggestion
description: Install Flutter and get started. Downloads available for Windows, macOS, Linux, and Chrome OS operating systems.
```

File: src/docs/index.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 title: Flutter documentation
 short-title: Docs
-description: The landing page for Flutter documentation.
+description: Get started with Flutter. Widgets, examples, updates and API docs to write your first Flutter app.
```
Comment (kwalrath) [inline src/docs/index.md:4]:
```suggestion
description: Get started with Flutter. Widgets, examples, updates, and API docs to help you write your first Flutter app.
```

**Review by kwalrath — APPROVED** [review]

**Comment by kwalrath** [comment]
Did Tim provide these? They're missing some serial commas. :)

**Comment by sfshaza2** [comment]
LOL. No, they came from a social media guy that Anjan is working with.

**Pull request merged**